packages/client-python/src/rocketride/core/runtime/resolver.py

```
@@ -0,0 +1,149 @@
+            is_prerelease = '-prerelease' in version_str
+
+            if version_spec == 'prerelease' and is_prerelease:
+                try:
+                    candidates.append(version_str)
+                except Exception:
+                    continue
+            elif version_spec == 'latest' and not is_prerelease:
+                try:
+                    candidates.append(version_str)
+                except Exception:
+                    continue
```

Ruff (0.15.7)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)